ivy/functional/frontends/numpy/linalg/matrix_and_vector_products.py

```diff
@@ -56,3 +56,9 @@ def tensorsolve(a, b, axes=2):
 def kron(a, b):
     a, b = promote_types_of_numpy_inputs(a, b)
     return ivy.kron(a, b)
+
+@handle_numpy_out
```

Comment:
Test fails for torch because float16 is not supported, please add: `@with_unsupported_dtypes({"2.0.0 and below": ("float16",)}, "torch")`